public/app/features/explore/TraceView/components/common/NewWindowIcon.tsx

```diff
@@ -32,13 +32,8 @@ type Props = {
   className?: string;
 };
 
-export default function NewWindowIcon(props: Props) {
-  const { isLarge, className, ...rest } = props;
+export default function NewWindowIcon({ isLarge = false, className }: Props) {
   const styles = useStyles2(getStyles);
   const cls = cx({ [styles.NewWindowIconLarge]: isLarge }, className);
-  return <Icon className={cls} name={'anchor'} {...rest} />;
```

Comment:
Should we also be getting rid of the destructing of the ...rest of the props or is there more other props of interest to underlying component tree?

Reply:
same here really, there are no other props listed in the type interface, so i don't think this is doing anything